packages/ui5-application-inquirer/src/prompts/prompt-helpers.ts

```diff
@@ -23,9 +23,9 @@ export function appPathExists(appName: string, targetPath?: string): boolean | s
  */
 export function defaultAppName(targetPath: string): string {
     let defProjNum = defaultProjectNumber;
-    let defaultName = t('prompts.appNameDefault');
+    let defaultName = t('prompts.main.name.default');
```

Comment:
Dont think this additional subgroup is necessary. At the end theres one set of prompts

Reply:
I've removed the `main` and `advanced` from the nested json